src/subscription_manager/managercli.py

```diff
@@ -1082,11 +1082,21 @@ def _get_org(org):
     @property
     def org(self):
         if not self._org:
-            owners = self.cp.getOwnerList(self.options.username)
-            if len(owners) == 1:
-                self._org = owners[0]['key']
+            if self.options.org is not None:
+                self._org = self.options.org
             else:
-                self._org = self._get_org(self.options.org)
+                owners = self.cp.getOwnerList(self.options.username)
+                if len(owners) == 0:
+                    system_exit(
+                        os.EX_DATAERR,
+                        _("Error: User {username} is not member of any organization.").format(
+                            username=self.options.username
+                        )
+                    )
+                elif len(owners) == 1:
+                    self._org = owners[0]['key']
+                else:
```

Comment:
Why did you exclude following code from this backport PR?

```python
# Get a list of valid owners. Since no owner was specified,
# print a hint message showing available owners, before asking
# to enter one.
org_keys = [owner['key'] for owner in owners]
print(_(
    'Hint: User "{name}" is member of following organizations: {orgs}'
).format(name=self.username, orgs=', '.join(org_keys)))
```

Reply:
Because this was a separate change (adding hints on available organization) that IIRC we decided to have only in main for now.

Reply:
I strongly believe that our customers want to have this functionality in rhel8 too. You can add it as another commit.

Reply:
Done; I backported the commits that did that as they were, as it was simply easier to apply them.